src/main/scala/mesosphere/marathon/core/launchqueue/LaunchQueue.scala

```diff
@@ -24,6 +24,7 @@ object LaunchQueue {
       tasksLaunchedOrRunning: Int,
       backOffUntil: Timestamp) {
     def waiting: Boolean = tasksLeftToLaunch != 0 || taskLaunchesInFlight != 0
+    def launchedOrRunning: Int = tasksLaunchedOrRunning + taskLaunchesInFlight
```

Comment:
Now there is a val `tasksLaunchedOrRunning` and a def `launchedOrRunning` – either use unambiguous names or get rid of one of them.